pkg/controller/healthz.go

```diff
@@ -26,14 +26,14 @@ func HandleHealthz(hctx context.Context, h *render.Renderer, cfg *database.Confi
 				db, err := cfg.Open(ctx)
 				if err != nil {
 					logger.Errorw("database error", "error", err)
-					h.JSON500(w, h.InternalError)
+					h.JSON500(w, render.InternalError)
```

Comment:
Sorry, I meant `controller.InternalError(w, r, h)`

Reply:
In dev mode, that'll actually render the error. In prod, it'll be a generic 500 page.

Reply:
@icco ^

Reply:
Done.